### PR TITLE
fix: Modified event type public api hander

### PR DESCRIPTION
**Pull request by Nikeshkumar-tk**
## What does this PR do?

 Get event-type handler will throw an http error  of status code 404 if no event types are found  

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ x] Verify wheather public api get event types is giving a response of status code 404, if  no event types are found.

## Mandatory Tasks
- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/api/pages/api/event-types/_get.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ async function getHandler(req: NextApiRequest) {
       owner: { select: { username: true, id: true } },
     },
   });
+  if(data.length === 0)  new HttpError({ statusCode: 404, message: "No event types were found" });
```
Comment (keithwillcode) [inline apps/api/pages/api/event-types/_get.ts:59]: @Nikeshkumar-tk Can you fix the spacing here? 
- Add a space between `if` and `(`.
- Remove 1 of the 2 spaces before `new`

**Review by keithwillcode — DISMISSED** [review]

File: apps/api/pages/api/event-types/_get.ts (hunk 1 above)
Comment (leog) [inline apps/api/pages/api/event-types/_get.ts:59]:
```suggestion
  if (data.length === 0) new HttpError({ statusCode: 404, message: "No event types were found" });
```

Comment (leog) [reply apps/api/pages/api/event-types/_get.ts:59]:
```suggestion
  if (data.length === 0) new HttpError({ statusCode: 404, message: "No event types were found" });
```

**Review by leog — APPROVED** [review]
LGTM, thanks

**Pull request merged**